build.gradle

```diff
@@ -100,15 +98,16 @@ repositories {
 dependencies {
     compile group: 'org.radarcns', name: 'radar-commons', version: radarCommonsVersion
     compile group: 'org.apache.avro', name: 'avro', version: avroVersion
-    compile (group: 'org.radarcns', name: 'radar-commons-testing', version: radarCommonsVersion) {
-        exclude group: 'org.slf4j', module: 'slf4j-simple'
-    }
+    compile group: 'org.radarcns', name: 'radar-commons-testing', version: radarCommonsVersion
     compile group: 'org.radarcns', name: 'radar-schemas-commons', version: radarSchemasVersion
 
     // Kafka streaming API
     compile group: 'org.apache.kafka', name: 'kafka-streams', version: kafkaVersion
-    compile (group: 'io.confluent', name: 'kafka-avro-serializer', version: confluentVersion) {
+    compile (group: 'io.confluent', name: 'kafka-streams-avro-serde', version: confluentVersion) {
         exclude group: 'org.slf4j', module: 'slf4j-log4j12'
+        exclude group: 'log4j', module: 'log4j'
+        exclude group: 'jline', module: 'jline'
+        exclude group: 'io.netty', module: 'netty'
     }
 
```

Comment:
can we also add `compile group: 'org.slf4j', name: 'slf4j-simple', version: slf4jVersion`. Logging doesn't seem to work without it.

Reply:
Logging should be enabled if you add `-Dlog4j.configuration=log4j.properties` to the command-line in docker, is that the case? Otherwise, we can also distribute this as an application, like with `distTar`. I've made a commit to that effect. Then a file `build/distributions/radar-backend-0.1.tar.gz` is uploaded to GitHub. When untarred, it contains a `radar-backend-0.1/bin/radar-backend` executable which already contains the logging property. See the `kafka-radarinit` `Dockerfile` for how to unpack and use a distribution like that.